Tools/CMake/AMReXOptions.cmake

```
@@ -321,6 +321,10 @@ cmake_dependent_option(AMReX_PROFPARSER "Enable profile parser" OFF
    "AMReX_BASE_PROFILE;AMReX_TRACE_PROFILE;AMReX_AMRDATA" OFF)
 print_option( AMReX_PROFPARSER )
 
+cmake_dependent_option(AMReX_ROCTX  "Enable roctx markup for HIP with ROCm" OFF
+     "AMReX_GPU_BACKEND STREQUAL HIP" ON)
```

This default looks a bit wrong to me.
Do you try to turn it on by default if HIP is used?
Ref: https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html
```suggestion
cmake_dependent_option(AMReX_ROCTX  "Enable roctx markup for HIP with ROCm" ON
     "AMReX_GPU_BACKEND STREQUAL HIP" OFF)
```

If you want this to be OFF by default and _OFF and hidden_ even if HIP is not found, then this:
```suggestion
cmake_dependent_option(AMReX_ROCTX  "Enable roctx markup for HIP with ROCm" OFF
     "AMReX_GPU_BACKEND STREQUAL HIP" OFF)
```

---

I believe it should be the second one, which is consistent with the gnu make system.